Feature/prefect workflow

chiaweh2 opened the pull request:
- introduce autmomatic dataset scan to add new dataset into TDS server
- add data adding log for better tracking
- batch dataset adding 397 datasets

chiaweh2 committed:
add . folder

chiaweh2 committed:
chnage env variable names

chiaweh2 committed:
add docstring and improve logging

chiaweh2 committed:
include data adding log order for better tracking

chiaweh2 committed:
clean up added filter option

chiaweh2 committed:
add missing data in first batch

rpconroy reviewed (commented):
This looks great! Thanks for adding this in. 

One quick question: Do you know the reason behind these wildcards:
```
        <exclude wildcard=".*" />
        <exclude wildcard=".*/" />
```

I could guess that the second one excludes directories, but I'm not understanding the first.

rpconroy reviewed (approved):
Looks good. Feel free to merge whenever.

chiaweh2 committed:
add missing info in the original xml files

chiaweh2 committed:
add check dataset.xml existance logic

skip if exist and need manual check. This avoids overwritting existing dataset xml file
